bookkeeper-server/src/main/java/org/apache/bookkeeper/util/DoubleByteBuf.java

```diff
@@ -280,6 +281,56 @@ public int setBytes(int index, ScatteringByteChannel in, int length) throws IOEx
     }
 
     @Override
+    protected short _getShortLE(int index) {
+        throw new UnsupportedOperationException();
```

Comment:
What is the impact of not implementing this interface? Or is it required for the new netty version?